src/propsheet/fontdlg.cpp

```diff
@@ -1090,7 +1095,10 @@ FindCreateFont(
                 Size = DefaultFontSize;
             }
         } else {
-            MakeAltRasterFont(CodePage, &AltFontSize, &AltFontFamily, &AltFontIndex, AltFaceName);
+            if (!NT_SUCCESS(MakeAltRasterFont(CodePage, &AltFontSize, &AltFontFamily, &AltFontIndex, AltFaceName)))
+            {
+                goto fallback;

```

Comment:
Is there a way to structure this without the `goto`? We try to avoid it when possible.

Reply:
Sure.